src/graphql.js

```diff
@@ -61,12 +61,7 @@ function buildFundingEligibilityVariables() : FundingEligibilityParams {
     };
 }
 
-function getDefaultVariables<V>() : V {
-    // $FlowFixMe[incompatible-return]
-    return {};
-}
-
-export function callGraphQL<T, V>({ query, variables = getDefaultVariables(), headers = {} } : {| query : string, variables : V, headers? : { [string] : string } |}) : ZalgoPromise<T> {
+export function callGraphQL<T, V = {}>({ query, variables, headers = {} } : {| query : string, variables : V, headers? : { [string] : string } |}) : ZalgoPromise<T> {
```

Comment:
Let's maintain the existing behavior where `variables` defaults to an empty object.
```suggestion
export function callGraphQL<T, V = {}>({ query, variables = {}, headers = {} } : {| query : string, variables : V, headers? : { [string] : string } |}) : ZalgoPromise<T> {
```

Reply:
Hey @gregjopa, I wonder if the `V` type is really a generic type, it seems to me that is a specific type, GraphQl defines how to call the endpoint not the logic, meaning the variables should have a format as well the query argument.
Maybe I'm wrong but I don't see variables as a generic type, we should compliance the GraphQl format.